Sources/AppStoreConnectCLI/Services/AppStoreConnectService.swift

```diff
@@ -139,6 +139,46 @@ class AppStoreConnectService {
             .await()
             .map(BetaTester.init)
     }
+
+    func removeTesterFromGroups(email: String, groupNames: [String]) throws {
+        let testerId = try GetBetaTesterOperation(
+                options: .init(id: nil, email: email)
+            )
+            .execute(with: requestor)
+            .await()
+            .betaTester
+            .id
+
+        let groupIds = try groupNames.map { try betaGroupIdentifier(matching: $0).await() }
+
+        try RemoveTesterOperation(options:
+                .init(removeStrategy:
+                    .removeTesterFromGroups(testerId: testerId, groupIds: groupIds)
+                )
+            )
+            .execute(with: requestor)
+            .await()
```

Comment:
formatting looking a little odd here:

```suggestion
        let operation = RemoveTesterOperation(
            options: .init(
                removeStrategy: .removeTesterFromGroups(testerId: testerId, groupIds: groupIds)
            )
        )
        
        try operation.execute(with: requestor).await()
```